lib/matplotlib/animation.py

```diff
@@ -1111,10 +1121,15 @@ class to use, such as 'ffmpeg'. If ``None``, defaults to
                 for anim in all_anim:
                     # Clear the initial frame
                     anim._init_draw()
+                frame_number = 0
+                total_frames = sum([a.save_count for a in all_anim])
```

Comment:
I'm not sure `a.save_count` is safe; right now if passed an iterator with no known length, `save_count` is truncated to 100 frames, but that behaviour is deprecated and supposed to go away. Then we will not have `save_count`...

Reply:
I see your point. But since I do not know, which mechanism will finally replace `save_count` it is at the moment difficult to make good decision. Any suggestions?

Reply:
Should we postpone this decision to the point, when save_count will be replaced?

Reply:
@QuLogic I know it's your free time, but could you please comment on that?

Reply:
IMO we can rely on `save_count` as long as it's there. In future versions it may become `None`, because there are situations where we do not know the number of frames. It may be reasonable to already support that as a possible outcome, even if the current animation code does prevent this using fixed limits in such a case.